frontend-react/src/pages/submissions/SubmissionDetails.tsx

```diff
@@ -127,8 +127,16 @@ function SubmissionDetailsContent() {
 */
 function SubmissionDetails() {
     const { actionId } = useParams<SubmissionDetailsProps>();
+    const location = useLocation();
+    const submissionCrumb = { label: "Submissions", path: "/submissions" };
+    const messageSearchCrumb = {
+        label: "Message Tracker",
+        path: "/admin/message-tracker",
+    };
     const crumbs: CrumbConfig[] = [
-        { label: "Submissions", path: "/submissions" },
+        (location.state as any).from === "/admin/message-tracker"
+            ? messageSearchCrumb
+            : submissionCrumb,
```

Comment:
From what I've seen in design, we almost certainly have a more generalized need for this behavior. Can we work this into the `Crumbs` component so any use of Breadcrumbs could utilize it?

An immediate benefit would be navigation between various resource pages in our `Product`, `Resources` and `Support` public pages. Right now, their crumbs are hard-coded to go back to the index of each particular page. So, in the case of navigating from `index a > page a` to `index b > page b` your backwards crumb will not take you back to `page a`, it'll take you to `index b`, and then you're lost or reliant on the browser back button.

I'll leave it to @stephenkao and you to determine if it's in/out of scope here but flagging for posterity!

Reply:
Hmmmm actually, this might be a question for @jillian-hammer and @chris-kuryak, but I think there are two separate concepts here:

- Breadcrumbs, which are named absolute links to illustrate website taxonomy
- Back button, which are arbitrary links to return to the point of origin

Clicking on a breadcrumb that says X should always go to X (hard link) whereas clicking on a back button should return you to where you just were (pop from history).

Like GitHub's file explorer is an example of breadcrumbs:
<img width="519" alt="Screenshot 2023-01-05 at 11 02 32" src="https://user-images.githubusercontent.com/2421042/210825541-7316562c-71f3-4371-935d-2181b653d06f.png">
(Clicking on "styles" will always navigate to the "styles" directory in GitHub regardless of where you navigated from.)

In this particular case, I think we're trying to force a breadcrumb to be a back button -- so why not just change this to be a back button (with a dynamic "from" path)?

Reply:
Hmm, I think my bias is towards trialing back buttons! But maybe we could consult Insights about it 👀

Reply:
Stephen brings up a good point. If we were to stick to the breadcrumb philosophy we would create a new route that goes /message-tracker/:submissionid which would then have a static breadcrumb back to message-tracker rather than going into submission's subroute.

Reply:
Let's bring this up on the ticket to get feedback from Jillian and Chris!